install/installing-with-kata-manager.md

```diff
@@ -0,0 +1,37 @@
+# Installing with kata-manager
+
+* [Introduction](#Introduction)
+* [Full Installation](#full-installation)
+* [Install the Kata packages only](#install-the-kata-packages-only)
+* [Further Information](#further-information)
+
+## Introduction
+`kata-manager` automates the Kata Containers installation procedure documented for [these Linux distributions](README.md#supported-distributions).
+
+> Note:
+> - Full installation mode is only available for Docker container manager. For other setups, you
+> can still use `kata-manager` to [install Kata package](#install-kata-packages-only), and then setup your container manager manually.
+
+## Full Installation
+This command will:
+1. install Kata Containers packages,
+2. install Docker,
+3. configure Docker to use the kata OCI runtime by default.
+
+```
+$ bash -c "$(curl -fsSL \
+$ https://raw.githubusercontent.com/kata-containers/tests/master/cmd/kata-manager/kata-manager.sh) \
+ install-docker-system"
+```
+
+## Install the Kata packages only
+This command will only install Kata Containers packages.
+
+```
+$ bash -c "$(curl -fsSL \
+$ https://raw.githubusercontent.com/kata-containers/tests/master/cmd/kata-manager/kata-manager.sh) \
+ install-packages"
+```
+
+## Further Information
+`kata-manager` can do much more; refer to [`kata-manager` page](https://github.com/kata-containers/tests/blob/master/cmd/kata-manager) for more information.
```

Comment:
Suggested rewrite: For more information on what `kata-manager` can do, refer to the [`kata-manager` page](https://github.com/kata-containers/tests/blob/master/cmd/kata-manager).